updateinfo/updateinfo-permissive.xsd

```diff
@@ -100,6 +100,14 @@
 	    addition of a single element (reboot_suggested), and thus taking
 	    the second occurrence of the update element for a specific id as the
 	    canonical one would ensure correctness.
+
+            openSuSE 11 used the source package name as the update ID. In this
```

Comment:
`openSUSE`

Reply:
fixed (and in another place it was there)